libs/blog/layouts/ui-navigation/src/lib/navigation/navigation.component.html

```diff
@@ -2,15 +2,16 @@
   class="flex flex-wrap"
   [ngClass]="{
     'flex-row': layout() === 'horizontal',
-    'flex-col': layout() === 'vertical'
+    'flex-col': layout() === 'vertical',
   }"
 >
   @for (item of navItems; track $index) {
     <a
       class="text-al-foreground p-2 font-medium md:p-6"
       [routerLinkActive]="'text-al-pink'"
-      [routerLink]="item.link | localize"
+      [href]="item.link | localize"
```

Comment:
using href instead of routerLink will cause reloading angular.
you can add optional `externalLink` field to the interface and create a condition to use href if that property is defined

Reply:
Corrected